src/test/scala/cromwell/MainSpec.scala

```diff
@@ -2,7 +2,7 @@ package cromwell
 
 import java.io.File
 
-import cromwell.binding.WdlBinding
+import cromwell.binding.WdlNamespace$
```

Comment:
Bleh, IntelliJ refactoring put a lot of these in there for some reason, and I had to hunt them all out.  I left a straggler I guess.